usage-guides/api-keys.md

```diff
@@ -0,0 +1,192 @@
+- If failed: Verify key and secret are correct
+
+### Sync Settings:
+
+- Option to enable/disable exchange synchronization
+- Consider disabling to avoid IP bans from frequent syncs
+
+> **Note**: At the moment, [margin trades](https://github.com/rotki/rotki/issues/1980) and [future trades](https://github.com/rotki/rotki/issues/1606) are not yet supported in rotki.
+
+### API key permissions
+
+rotki only needs read-only permissions for your accounts. As a general rule, exchanges (e.g., Binance, Coinbase Pro) group all the read-only permissions as "read" or "view".
+
+![Simple API key permissions](/images/add_exchange_api_keys_binance.png)
+
+In the case of an exchange providing a more granular permissions scheme (e.g., Coinbase, Kraken) or having additional options (e.g., query limits, passphrase), refer to the exchange documentation or get in touch via their customer support channel.
+
+![Granular API key permissions](/images/add_exchange_api_keys_coinbase.png)
+
+You may also try creating an API key with the minimum read-related permissions, then adding it in rotki and finally checking that the connection was successful and data was loaded as expected. Otherwise, try again adding more read-related permissions.
+
+#### Kraken
+
+When inputting the API key for Kraken, you need to specify the type of your Kraken account, which depends on your Kraken account verification level. Refer to [this](https://support.kraken.com/hc/en-us/articles/360001395743-Verification-levels-explained) for more information.
+
+![Kraken account type](/images/exchanges_add_kraken.png)
+
+#### Binance / Binance US
+
+To improve the speed of querying trade information using the Binance API, you can specify which market pairs to query instead of querying all possible pairs. This reduces the number of requests made to Binance servers, avoiding potential rate limits and failures. To select specific markets, edit your Binance exchange instance configuration.
+
+![Edit Binance in the exchanges section](/images/exchanges_edit_binance.png)
+
+And choose the markets in the `Filter market pair(s)` search.
+
+![Binance markets selection](/images/binance_markets_selection.png)
+
+Once finished, click on save.
+
+## External services
+
+### Why External API Keys Are Needed
+
+Required for accessing:
+
+- Historical crypto prices
+- EVM transactions
+- Other external data services
+
+### How to External Add API Keys:
+
+1. Create free account on service website
+2. Generate API key
+3. In rotki:
+   - Go to `API Keys → External Services`
+   - Enter your API key(s)
+
+![External services](/images/external_services.png)
+
+### Etherscan
+
+At the moment, there is no compulsory API key. But if you don't use your own node, Etherscan queries without an API key are really slow. So if you don't already have an account with them, please create one [here](https://etherscan.io/register) and then generate a free API key and set it in the app as explained above. It's free of charge.
```

Comment:
it is kind of mandatory now. Check the info we have but I believe it is required now and we have this in the popup when adding the address

Reply:
It's still working I think without etherscan key? The popup when adding the address, it's only a warning / info.

Reply:
I am also under the impression that it can still work, but it is really slow. Maybe it would make sense to strongly suggest the api key usage, just because of the fact that it improves user experience?

Reply:
Updated 
<img width="704" alt="image" src="https://github.com/user-attachments/assets/59c76d3a-cb70-418d-aad3-c75e9a3d6929" />